modules/PropertyManagement/Database/Migrations/2019_07_22_000400_create_Realty_table.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+
+class CreateRealtyTable extends BaseMigration
+{
+    protected $tablename = 'realty';
+
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create($this->tablename, function (Blueprint $table) {
+            $this->up_table_generic($table);
+
+            $table->integer('node_id')->nullable();
+
+            $table->string('name')->nullable();
+            $table->string('number')->nullable();
+
+            $table->string('street');
+            $table->string('house_nr');
+            $table->string('zip');
+            $table->string('city');
+
+            $table->string('administration')->nullable();
+            $table->string('expansion_degree')->nullable();
+            $table->string('concession_agreement')->nullable();
+            $table->date('agreement_from')->nullable();
+            $table->date('agreement_to')->nullable();
+
+            $table->date('last_restoration')->nullable();
```

Comment:
personal opinion: would add an indicator for a date field (_at or timestamp _on for date)
```suggestion
            $table->date('last_restoration_on')->nullable();
```

Reply:
applied